components/contribution-flow/index.js

```diff
@@ -641,7 +641,12 @@ class ContributionFlow extends React.Component {
             return true;
           } else {
             const isCompleted = Boolean(noPaymentRequired || stepPayment);
-            if (isCaptchaEnabled() && !stepProfile.captcha && !LoggedInUser) {
+            if (
+              !stepProfile.captcha &&
+              isCaptchaEnabled() &&
+              !LoggedInUser &&
+              stepPayment?.key === NEW_CREDIT_CARD_KEY
```

Comment:
That looks like for new Credit Card only? Are we not enforcing for existing Credit Cards?

Reply:
Exactly. Considering captcha is only required for guest accounts, I think this will work.